pubspec.yaml

```diff
@@ -27,6 +27,9 @@ dependencies:
   firebase_analytics: ^6.2.0
   firebase_auth: ^0.18.3
   cloud_firestore: ^0.14.3
+  flutter_spinkit: ^4.0.0
+  provider: ^3.1.0
```

Comment:
Extra dependencies that we will most likely not use in the end.

Reply:
how to implement what I've implemented w/o the provider?